Assets/MixedRealityToolkit.Tests/EditModeTests/Experimental/MigrationToolTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Experimental.UI;
+using Microsoft.MixedReality.Toolkit.Experimental.Utilities;
+using Microsoft.MixedReality.Toolkit.UI;
+using NUnit.Framework;
+using System;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class MigrationToolTest
```

Comment:
```suggestion
    public class MigrationToolTests
```
I think this is the standard. Same for the file name.

Reply:
Changing